dev/io.openliberty.microprofile.openapi.4.0.internal_fat/fat/src/io/openliberty/microprofile/openapi40/fat/FATSuite.java

```diff
@@ -33,4 +35,14 @@
                 VersionWarningTest.class,
 })
 @RunWith(Suite.class)
-public class FATSuite {}
+public class FATSuite {
+
+    public static RepeatTests defaultRepeat(String serverName) {
+        return MicroProfileActions.repeat(serverName,
```

Comment:
```suggestion
        return MicroProfileActions.repeat(serverName, TestMode.FULL, true,
```
This way you don't need the transformer.

Reply:
I'm going to swap out that boolean parameter to make the meaning more clear when it's used in a test like this.

Reply:
Raised #31324